rdfjs-high-level-spec.md

```diff
@@ -0,0 +1,94 @@
+# RDF/JS High Level API
+
+## Mission Statement
+
+**Status:** This document is a _draft_.<br>
+
+## Data interfaces
+
+### Dataset
+
+**Important:** Unlike Arrays, RDF Datasets have the following properties:
+
+* they are un-ordered (and quads within them cannot be accessed using `[]` notation)
+* like Sets, they are unique (duplicate identical quads are not allowed)
+
+**Properties:**
+
+- `number .length` Number of quads in the dataset. (Similar to `Array.length`) 
+
+**Static/Factory methods:**
+
+- `static Promise<Dataset> import (Function factory, Stream stream)`
```

Comment:
I think it forwards data factory which dataset should use https://github.com/rdfjs/representation-task-force/issues/79